mybatis.properties

```diff
@@ -0,0 +1,20 @@
+category=External Analysers
+description=Rules to check SQL statements in MyBatis Mapper XML files.
+homepageUrl=https://github.com/donhui/sonar-mybatis
+archivedVersions=1.0.0
+publicVersions=1.0.1
+
+defaults.mavenGroupId=org.sonarsource.plugins.mybatis
+defaults.mavenArtifactId=sonar-mybatis-plugin
+
+1.0.1.description=update the descrption of rules and document
+1.0.1.sqVersions=[6.7,LATEST]
+1.0.1.date=2019-08-13
+1.0.1.changelogUrl=https://github.com/donhui/sonar-mybatis/releases/tag/1.0.1
+1.0.1.downloadUrl=https://github.com/donhui/sonar-mybatis/releases/download/1.0.1/sonar-mybatis-plugin-1.0.1.jar
+
+1.0.0.description=Initial Update Center release
+1.0.0.sqVersions=[6.7,LATEST]
```

Comment:
Only one version can be compatible with `LATEST`. In fact, since version 1.0.0 hasn't yet made it into the Marketplace, there's no reason to keep it in this file & send it straight to 'archived'. I suggest you just delete this version.

Reply:
OK, I have removed version 1.0.0.